src/types/guild.ts

```diff
@@ -660,3 +666,40 @@ export interface EditGuildTemplate {
   /** description for the template (0-120 characters) */
   description?: string | null;
 }
+
+/** */
+export interface MembershipScreeningPayload {
+  /** when the fields were last updated */
+  version: string;
```

Comment:
are u sure? name sounds like it would be a verion sting not a last updated at

Reply:
Reference: [Membership Screening PR](https://github.com/discord/discord-api-docs/pull/2396/files#diff-6a43fc10e10de61f31b90ab22421ef0612d0ea75e3cd53891c6393a6fb84872dR370)